site/pages/components/tag.tsx

```diff
@@ -0,0 +1,564 @@
+      'It can take one space token to specify the logical block start and end padding of the container. This space token can also be used on breakpoints.',
+  },
+  {
+    attribute: 'paddingBlockStart',
+    type: 'MQ<string>',
+    description:
+      'It can take one space token to specify the logical block start padding of the container. This space token can also be used on breakpoints.',
+  },
+  {
+    attribute: 'paddingBlockEnd',
+    type: 'MQ<string>',
+    description:
+      'It can take one space token to specify the logical block end padding of the container. This space token can also be used on breakpoints.',
+  },
+];
+
+const TagComponent = (layoutProps: LayoutProps) => (
+  <ComponentPageTemplate
+    headTags={{
+      title: 'Tag',
+      description:
+        'Tags are used to classify content, typically using keywords.',
+    }}
+    layoutProps={layoutProps}
+    pageIntroduction={{
+      type: 'Navigation',
+      name: 'Tag',
+      hero: {
+        illustration: 'components/tag-illustration',
+      },
+      introduction:
+        'Tags are used to classify content, typically using keywords.',
+    }}
+    componentDefaultsKey="tag"
+    meta={{
+      status: MetaStatus.Supported,
+      introduced: 'v0.2.0',
+      codeUrl: 'https://github.com/newscorp-ghfb/newskit/tree/main/src/tag',
+      figmaUrl:
+        'https://www.figma.com/file/FSbCQa6SzVR3K48ZWLeD77/%F0%9F%9F%A2-NK-Web-Components?node-id=1952%3A3407',
+    }}
+    interactiveDemo={{
+      introduction:
+        'This demo allows you to preview the tag component, its variations, and configuration options.',
+      playground: {
+        componentName: 'Tag',
+        component: state => <Tag {...state} />,
+        knobs: [
+          {
+            name: 'Content',
+            propName: 'children',
+            value: 'Tag Content',
+          },
+          {
+            name: 'Link',
+            propName: 'href',
+            value: 'https://www.newskit.co.uk/',
+          },
+          {
+            name: 'Disabled',
+            propName: 'disabled',
+            type: 'boolean',
+            value: false,
+          },
+          {
+            name: 'Tag Size',
+            propName: 'size',
+            options: [
+              {
+                label: 'small',
+                value: 'small',
+              },
+              {
+                label: 'medium',
+                value: 'medium',
+                isDefault: true,
+              },
+              {
+                label: 'large',
+                value: 'large',
+              },
+            ],
+          },
+          {
+            name: 'Overrides',
+            propName: 'overrides',
+            options: [
+              {
+                label: 'Default',
+                value: undefined,
+                isDefault: true,
+              },
+              {
+                label: 'Style Preset',
+                value: {
+                  stylePreset: 'tagCustom',
```

Comment:
tagCustom does not exist try with:
```suggestion
                  stylePreset: 'tagPrimaryInverse',
```

Reply:
Done. Modified the stylePreset value. -thanks!